fitbenchmarking/fitbenchmarking
# Command line option for tests

#### Description of Work

Fixes #546 

Adds command line option to allow tests to be run with default installation.

#### Testing Instructions

1. Check all automated travis tests pass

Function: Does the change do what it's supposed to?

Tests: Does it pass? Is there adequate coverage for new code?

Style: Is the coding style consistent? Is anything overly confusing?

Documentation: Is there a suitable change to documentation for this change?
